### PR TITLE
feat(azure): Adds permission desc for UAMI BPs

### PR DESCRIPTION
**Pull request by tfelix**
Concerns: CU-96xdyn

### CODE REVIEW

**Comment by meshcloud-gmbh** [comment]
Task linked: [CU-96xdyn Azure Blueprint Replication needs additional role](https://app.clickup.com/t/96xdyn)

**Review by felixzieger — APPROVED** [review]

File: docs/meshstack.azure.landing-zones.md [hunk 1]
```diff
@@ -91,7 +91,10 @@ This flag allows you to use a User Assigned Managed Identity (UAMI) instead of t
 
 Currently we only recommend to use this flag if you want to create resources that require permissions outside of the meshcloud managed subscription. A good example is if you want to connect a meshcloud managed subscription to a central log workspace. In order to do this, you require permissions in both the meshcloud managed subscription and the subscription your central Log Workspace App resides in.
 
-**Please note**: meshcloud automatically handles permissions in the subscription the Blueprint is assigned to, but permissions outside of that cannot be handled and have to be setup by yourself.
+>**Please note**: meshStack can not handle permissions outside of the Subscriptions it manages. You need to make sure the Managed Identity has the correct permissions set. You must also give the replicator service principal the following permission on the target resource the Managed Identity lives in (e.g. the Subscription of the Managed Identitiy).
```
Comment (chewbeccy) [inline docs/meshstack.azure.landing-zones.md:94]: what are the correct permissions for the managed identity?

Comment (felixzieger) [reply docs/meshstack.azure.landing-zones.md:94]: Could this be added to [https://docs.dev.meshcloud.io/docs/meshstack.azure.index.html#azure-rbac-permissions](https://docs.dev.meshcloud.io/docs/meshstack.azure.index.html#azure-rbac-permissions) or is this to specific to be added in the general setup of the replicator SPN?

**Comment by felixzieger** [comment]
Could this be added to [https://docs.dev.meshcloud.io/docs/meshstack.azure.index.html#azure-rbac-permissions](https://docs.dev.meshcloud.io/docs/meshstack.azure.index.html#azure-rbac-permissions) or is this to specific to be added in the general setup of the replicator SPN?

File: docs/meshstack.azure.landing-zones.md (hunk 1 above)
Comment (tfelix) [reply docs/meshstack.azure.landing-zones.md:94]: depends on what you want to do with the principal e.g. all what is required to assign your blueprint. But I can clarify on that.

**Comment by tfelix** [comment]
No this is not possible. This is only the permissions for our replicator principal. However, in order for this to work the replicator principal needs a special role on the target resource the Managed Identity lives in (e.g. the Subscription of the Managed Identitiy). This target resource is not known at the time we execute the terraform script. You probably have to assign this role manually whenever you create a new user managed identity.

File: docs/meshstack.azure.landing-zones.md (hunk 1 above)
Comment (tfelix) [reply docs/meshstack.azure.landing-zones.md]: You need to add the permissions the Blueprint requires to do its work. I clarify on that.

**Comment by felixzieger** [comment]
> No this is not possible. This is only the permissions for our replicator principal. However, in order for this to work the replicator principal needs a special role on the target resource the Managed Identity lives in (e.g. the Subscription of the Managed Identitiy). This target resource is not known at the time we execute the terraform script. You probably have to assign this role manually whenever you create a new user managed identity.

Thank you for pointing this out.

**Review by felixzieger — APPROVED** [review]

**Review by chewbeccy — APPROVED** [review]

**Pull request merged**